samples/tab-deeplink/csharp/assets/sample.json

```diff
@@ -3,10 +3,10 @@
     "name": "officedev-microsoft-teams-samples-tab-deeplink-csharp",
     "source": "officeDev",
     "title": "DeepLink",
-    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID",
+    "shortDescription": "Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.",
```

Comment:
It should be Microsoft Teams and not teams.

Reply:
Yes changes it